app/views/security_ruleset/_form.html.erb

```diff
@@ -4,7 +4,7 @@
     <label class="govuk-label" for="file-upload-1">
       Upload an OpenAPI document
     </label>
-    <%= form_tag("/ruleset", method: :post, multipart: true) do %>
+    <%= form_tag("/security_ruleset", method: :post, multipart: true) do %>
```

Comment:
Putting paths relative to root (i.e that start with "/") in rails rendered pages is problematic as you can't guarantee that the app will alway be mounted at root. Therefore it is alway better to use rails path helpers if they are available. So:
```ruby
<%= form_tag(security_ruleset_path, method: :post, multipart: true) do %>
```

Reply:
security_ruleset_path is not available as a helper method if you only have the new and create action. However you can use new_ security_ruleset_path as that has the correct url for a POST.

Reply:
This helper method is not available as you need the index action to generate that specific helper method.